scripts/modelbench/launch.sh

```diff
@@ -56,7 +56,9 @@ docker cp /home/ubuntu/docker/inductor_train.sh $USER:/workspace/pytorch
 docker cp /home/ubuntu/docker/version_collect.sh $USER:/workspace/pytorch
 
 if [ $TEST_MODE == "inference" ]; then
-    docker exec -i $USER bash -c "bash inductor_test.sh $THREADS $CHANNELS $PRECISION $TEST_SHAPE inductor_log $DYNAMO_BENCH $WRAPPER $HF_TOKEN"
+    docker exec -i $USER bash -c "bash inductor_test.sh $THREADS $CHANNELS $PRECISION $TEST_SHAPE inductor_log $DYNAMO_BENCH $WRAPPER $HF_TOKEN inference"
+elif [ $TEST_MODE == "training full scope" ]; then
+    docker exec -i $USER bash -c "bash inductor_test.sh $THREADS $CHANNELS $PRECISION $TEST_SHAPE inductor_log $DYNAMO_BENCH $WRAPPER $HF_TOKEN training"
```

Comment:
suggest to hardcode `THREADS` as `multiple` in training test

Reply:
Resolved